Common/Currencies.cs

```diff
@@ -85,7 +85,13 @@ public static class Currencies
             "EOSBTC",
             "XLMBTC",
             "ETCBTC",
-            "ZRXBTC"
+            "ZRXBTC",
+
+            //binance
+            "BTCUSDT",
+            "BCHUSDT",
+            "LTCUSDT",
```

Comment:
I wonder if we need this static data to be resolved on application start and/or periodically for long-running live algorithms

Reply:
Agreed. See my response above (https://github.com/QuantConnect/Lean/pull/3511#discussion_r316012763)